### PR TITLE
Update application-mysql.properties

### PR DESCRIPTION
**Pull request by Rognetta**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@Rognetta Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-petclinic&pullRequestId=506)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-petclinic&pullRequestId=506) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@Rognetta Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-petclinic&pullRequestId=506)!

**Comment by snicoll** [comment]
@Rognetta can you please explain the context and the purpose of this PR? Thank you.

File: src/main/resources/application-mysql.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # database init, supports mysql too
 database=mysql
-spring.datasource.url=jdbc:mysql://localhost/petclinic
-spring.datasource.username=root
-spring.datasource.password=petclinic
-# SQL is written to be idempotent so this is safe
+spring.datasource.url=${MYSQL_DB:jdbc:mysql://localhost/petclinic}
+spring.datasource.username=${MYSQL_USER:petclinic@petclinicdb2}
```
Comment (dsyer) [inline src/main/resources/application-mysql.properties]: Not sure we want the default to be `@anything`. The default should be fine without it.

File: src/main/resources/application-mysql.properties [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+spring.datasource.password=${MYSQL_PASS:petclinic}
+# Uncomment this the first time the app runs
```
Comment (dsyer) [inline src/main/resources/application-mysql.properties]: Why is this changed? Maybe an artifact of the PR. Rebase and revert this line?
